Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -330,6 +330,58 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
+#### Continuing or halting after test failures
+
+An instance of an `XCTestCase` subclass can set its
+[`continueAfterFailure`](https://developer.apple.com/documentation/xctest/xctestcase/1496260-continueafterfailure)
+property to `false` to cause a test to stop running after a failure occurs.
+XCTest stops an affected test method by throwing an Objective-C exception at the
```

Comment:
Minor note: 
```suggestion
XCTest stops an affected test by throwing an Objective-C exception at the
```
the failure may occur in a `setUp` or `tearDown`, so it's not strictly always the test _method_ which is halted. I think "stops an affected **test**" is bit more accurate.

Reply:
That's fair. I was trying to distinguish suites, which don't really have the same "presence" in XCTest as they do in swift-test.